backend/chat-backend/src/main/java/pl/wsb/chat/domain/message/Message.java

```diff
@@ -1,4 +1,56 @@
 package pl.wsb.chat.domain.message;
 
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.Date;
+
+@Document
 public class Message {
+    @Id
+    private ObjectId id;
+    private String notes;
+    private Date date;
```

Comment:
Preferable use LocalDateTime or Instant
https://programminghints.com/2017/05/still-using-java-util-date-dont/
util.Date is deprecated